Introduction to Kotlin Multiplatform/05_reuse_code.md

```diff
@@ -36,11 +36,11 @@ const val jvmPort = 8888
 The declarations from the Kotlin files under the
 `src/commonMain/kotlin` are visible to all targets. We can only
 use the platform-independent subset of the Kotlin standard library
-from these files. Kotlin Multiplatform libraries are allowed too.
+from Kotlin/Common source files. Kotlin Multiplatform libraries are allowed too.
 It allows one to easily share code between platforms.
 
-Now we may update the code in both `jsMain` and `jvmMain` source
-sets to use the newly added `jvmHost` and `jvmPort` constants.
+Let's now update the code in both `jsMain` and `jvmMain` source
```

Comment:
Let's now update the code in both the `jsMain` and `jvmMain` source
sets to use the newly added `jvmHost` and `jvmPort` constants